public-api/db-cdr/generate-cdr/run-achilles-queries.sh

```diff
@@ -1504,187 +1577,38 @@ where (o.observation_source_concept_id > 0 and o.value_source_concept_id = 0 and
-    (id, analysis_id, stratum_1, stratum_3, count_value, source_count_value)
-    with measurement_concepts as
-    (select measurement_concept_id as concept,co.person_id as person
-    from \`${BQ_PROJECT}.${BQ_DATASET}.measurement\` co join \`${BQ_PROJECT}.${BQ_DATASET}.concept\` c on co.measurement_concept_id=c.concept_id
-    where c.vocabulary_id != 'PPI' and co.measurement_concept_id not in (3036277,903118,903115,3025315,903135,903136,903126,903111,42528957)),
-    measurement_source_concepts as
-    (select measurement_source_concept_id as concept,co.person_id as person
-    from \`${BQ_PROJECT}.${BQ_DATASET}.measurement\` co join \`${BQ_PROJECT}.${BQ_DATASET}.concept\` c on co.measurement_source_concept_id=c.concept_id
-    where c.vocabulary_id != 'PPI' and co.measurement_source_concept_id not in (3036277,903133,903118,903115,3025315,903121,903135,903136,903126,903111,42528957,903120)
-    and co.measurement_source_concept_id not in (select distinct measurement_concept_id from \`${BQ_PROJECT}.${BQ_DATASET}.measurement\`)),
-    concepts as
-    (select * from measurement_concepts union all select * from measurement_source_concepts)
-    select 0 as id,3000 as analysis_id,'21' as stratum_1,'Measurement' as stratum_3, (select count(distinct person) from concepts) as count_value,
-    0 as source_count_value"
-
-    echo "Getting participant domain participant counts"
-    bq --quiet --project=$BQ_PROJECT query --nouse_legacy_sql \
-    "insert into \`${WORKBENCH_PROJECT}.${WORKBENCH_DATASET}.achilles_results\`
-    (id, analysis_id, stratum_1, stratum_3, count_value, source_count_value)
-    with procedure_concepts as
-    (select procedure_concept_id as concept,co.person_id as person
-    from \`${BQ_PROJECT}.${BQ_DATASET}.procedure_occurrence\` co join \`${BQ_PROJECT}.${BQ_DATASET}.concept\` c on co.procedure_concept_id=c.concept_id
-    where c.vocabulary_id != 'PPI' and c.concept_id not in (3036277,903118,903115,3025315,903135,903136,903126,903111,42528957)),
-    procedure_source_concepts as
-    (select procedure_source_concept_id as concept,co.person_id as person
-    from \`${BQ_PROJECT}.${BQ_DATASET}.procedure_occurrence\` co join \`${BQ_PROJECT}.${BQ_DATASET}.concept\` c on co.procedure_source_concept_id=c.concept_id
-    where c.vocabulary_id != 'PPI' and c.concept_id not in (3036277,903133,903118,903115,3025315,903121,903135,903136,903126,903111,42528957,903120)
-    and co.procedure_source_concept_id not in (select distinct procedure_concept_id from \`${BQ_PROJECT}.${BQ_DATASET}.procedure_occurrence\`)),
-    concepts as
-    (select * from procedure_concepts union all select * from procedure_source_concepts)
-    select 0 as id,3000 as analysis_id,'10' as stratum_1,'Procedure' as stratum_3, (select count(distinct person) from concepts) as count_value,
-    0 as source_count_value"
-fi
+### Test data does not have the mapping tables, so this else block lets the script to fetch domain counts for test data
```

Comment:
Ah, this is the comment I was looking for above, should this be moved as well?

Reply:
Moved the comment to the top of if block.